src/contributing.md

```diff
@@ -270,7 +270,7 @@ because the rust-lang/rust build won't pass until those tools build and pass the
 tests.
 
 That means that, in the default state, you can't update the compiler without first
```

Comment:
```suggestion
That would mean that, in the default state, you couldn't update the compiler without first
```
I still find this sentence very confusing, but now at least it is clearly non-factual.
(I also realize this is pre-existing.)

Reply:
Yes! I'm so glad I'm not the only one that was confused by this. It gave me a brief scare initially because I didn't realize until I'd gotten a few paragraphs in that it was actually describing a past or hypothetical state, but using present-tense prose. Applied in ed0c82bc91a5052d92a07580778759b00c537cc6